app/api/admin/route.js

```
@@ -0,0 +1,144 @@
+      
+      case 'resources':
+        data.resources = await Resource.find({}).sort({ createdAt: -1 });
+        break;
+      
+      case 'stats':
+        const [contactCount, applicationCount, registrationCount, eventCount] = await Promise.all([
+          Contact.countDocuments(),
+          Application.countDocuments(),
+          EventRegistration.countDocuments(),
+          Event.countDocuments()
+        ]);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)